components/Calendar/subcomponents/TooltipContent.js

```diff
@@ -0,0 +1,12 @@
+
+export default function AppointmentContent (e) {
```

Comment:
e is not descriptive as a prop name. Single name parameters and variables should only be used in a few instances, such as for loops and synthetic react events. Rename to props or something more descriptive

Reply:
On it!